docs/HowToArchiveJournal.md

```diff
@@ -17,13 +17,134 @@ For more info on NuoDB database journaling, please consult the official [NuoDB d
 
 ## Upgrading existing domains
-TODO
\ No newline at end of file
+
+### Overview
+
+StatefulSet volume mounts are immutable and any attempts to change this value will result in the following error:
+```
+Error: UPGRADE FAILED: cannot patch "sm-database-nuodb-cluster0-demo-hotcopy" with kind StatefulSet: \
+StatefulSet.apps "sm-database-nuodb-cluster0-demo-hotcopy" is invalid: spec: Forbidden: \
+updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden
+```
+
+To change the domain from using `an in-archive journal` to an `a separated journal` or vice versa, Kubernetes requires the StatefulSet to be deleted and recreated with the new settings.
+
+NuoDB Helm Charts offer two distinct Storage Manager StatefulSets.
+One StatefulSet controls Storage Managers which have a `backup` directory and are referred to as `HotCopy SMs`.
+The second StatefulSet which does not have such a directory, and we call Storage Managers in this SS `NoHotCopy SMs`.
+Since these StatefulSets can be upgraded independently, NuoDB will not suffer downtime or data loss when performing this upgrade operation.
+
+### Pre-requisites
+
+1) Your domain must contain an Enterprise License, and the ability to run two or more Storage Managers.
+2) You must have at least 1 Storage Manager in both `HotCopy SM` SS and the `NoHotCopy SM` SS.
+3) Your database option `max-lost-archives` is set to allow the loss of a whole StatefulSet.
+
+### Upgrade Process
+In this example, we will assume that the domain is running 1 Storage Manager in each StatefulSet and the name of the database is `demo` as follows:
+```shell
+kubectl get statefulsets.apps -n nuodb
+NAME                                      READY
+admin-nuodb-cluster0                      1/1
+sm-database-nuodb-cluster0-demo           1/1
+sm-database-nuodb-cluster0-demo-hotcopy   1/1
+```
+
+NOTE: NuoDB also requires the NuoDB admin layer, which is also a StatefulSet but won't be directly involved in this migration process.
+
+There will be a number of PersistentVolumeClaims.
+- 1 PVC for the NoHotCopy Storage Manager
+- 2 PVCs for the HotCopy Storage Manager
+- 1 PVC for the admin layer
+
+```shell
+k get persistentvolumeclaims -n nuodb
+NAME                                                       STATUS   VOLUME                                     CAPACITY   ACCESS MODES
+archive-volume-sm-database-nuodb-cluster0-demo-0           Bound    pvc-e8227428-6b9f-47b1-acc0-41b70a21d043   20Gi       RWO
+archive-volume-sm-database-nuodb-cluster0-demo-hotcopy-0   Bound    pvc-19540f97-8407-4fe1-a940-3b3f83538f0f   20Gi       RWO
+backup-volume-sm-database-nuodb-cluster0-demo-hotcopy-0    Bound    pvc-daebd030-a4eb-46e1-8451-6d65e4fa3061   20Gi       RWO
+raftlog-admin-nuodb-cluster0-0                             Bound    pvc-4930b80b-a622-487a-90a6-60a2bd3f0548   1Gi        RWO
+
+```
+
+
+#### Upgrade the NoHotCopy StatefulSet
+First, scale the StatefulSet to 0.
```

Comment:
Consider using number for your list as in other locations, rather than spelling out "First", "Second", etc.